docs/tools/platformsh.md

```diff
@@ -0,0 +1,27 @@
+# Platform.sh
+
+Platform.sh offers a command line tool to help . It enables you to do almost everything in a terminal that you can do in the Dashboard, and much more.
```

Comment:
command line too to help... ? Looks like it's missing something. To help development for platform.sh?

The text for Terminus states:
>Terminus is Pantheon's command line tool. It enables you to do almost everything in a terminal that you can do in the Dashboard, and much more.